lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -39,8 +39,7 @@
     .nx-checkbox__box, .nx-radio__focus {
       position: relative;
       &::after {
-        border: 1px solid var(--nx-color-interactive-border-focus);
-        box-shadow: var(--nx-box-shadow-focus);
+        border: 2px solid var(--nx-color-interactive-border-focus);
```

Comment:
I think we decided to use the `outline` property instead of a `border` for focus states.

Reply:
> I think we decided to use the `outline` property instead of a `border` for focus states.

I personally think it doesn't hurt to leave it here, since it's a circle and would look a bit odd with a square focus outline for those (very few) stubborn Safari users. But I'm open to other opinions!

Reply:
> I think we decided to use the `outline` property instead of a `border` for focus states.

I also think it might be a good case to leave it as a border here for the same reason Emily mentioned, let me know your thoughts!